docker/chromium/Dockerfile

```diff
@@ -1,8 +1,26 @@
-
-FROM demisto/python3:3.7.4.1150
+FROM demisto/python3-deb:3.7.4.1163
 
 COPY requirements.txt .
+COPY google-chrome-stable_current_amd64.deb .
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  gcc \
+  python-dev \
+  wget \
+  libxss1 \
+  libappindicator1 \
+  libindicator7 \
+&& pip3 install --no-cache-dir -r requirements.txt
+
+# Install Chrome
+#RUN wget https://dl.google.co/linux/direct/google-chrome-stable_current_amd64.deb; \
+RUN dpkg -i google-chrome*.deb; \
```

Comment:
Use `&&` instead of `;`

Reply:
> && will not run the next command on an exit code other than 0 (which is expected in this scenario)
@glicht

Reply:
ok. so use && for the next steps. and also add a verification that the install actually installed chrome. You can run something of the form: `/opt/google/chrome/chrome --version | grep 76.0.3809`

Reply:
> `&& python3 -c "from webdriver_manager.chrome import ChromeDriverManager; ChromeDriverManager().install()" \`

I added a python command that will ensure that chrome installed successfully, and will init the web driver so the

Reply:
Excellent.